Sming/SmingCore/Network/Http/Websocket/WebsocketConnection.h

```diff
@@ -121,9 +120,10 @@ class WebsocketConnection
 	void* getUserData();
 
 	// @deprecated
-	bool operator==(const WebsocketConnection& rhs) const;
+	bool operator==(const WebsocketConnection& rhs) const __deprecated;
```

Comment:
@slaff You deprecated this method on 04/05/2017, and enforcing deprecation means there's a corresponding change in the `HttpServer_WebSockets` sample and possibly user apps; Keep it deprecated?

Reply:
> You deprecated this method on 04/05/2017, and enforc ....

Hm... cannot remember why. You can un-deprecated if needed.